pom.xml

```
@@ -367,6 +372,13 @@
       <!-- Required for S3ProxyRule -->
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.7.0</version>
```

```suggestion
      <version>5.8.1</version>
```
Might as well go with the latest release.